docker/pypi/wmagent/bin/manage-common.sh

```diff
@@ -265,8 +275,22 @@ _status_of_oracle(){
 _renew_proxy(){
     # Auxiliary function to renew agent proxy
     local hostName=`hostname -f`
+    _load_wmasecrets
 
     # Here to find out if the agent is CERN or FNAL and use the proper credentials name for _renew_proxy
+    [[ "$TEAMNAME" == Tier0* ]] &&  {
+	echo "$FUNCNAME: This is a Tier0 agent"
+	local vomsproxyCmd="voms-proxy-init -rfc \
+	            -voms cms:/cms/Role=production -valid 168:00 -bits 2048 \
+		    -cert $X509_USER_CERT -key $X509_USER_KEY \
+                    -out  $X509_USER_PROXY"
+	$vomsproxyCmd || {
+	    echo "$FUNCNAME: ERROR: Failed to renew invalid myproxy"
+	    return $(false)
+	}
+    return
```

Comment:
Even though indentation does not matter in bash, it would be great to fix this return.

Reply:
because they were copiy-pasted from old Tier0 code, where somebody long long ago....  has used tabs the indentation, which is supper annoying, I admit. I'll fix it wherever I can find it, but if you notice some more, please point them as well.

Reply:
Todor, did you forget pushing your changes in? This whole block above is inside `[[ "$TEAMNAME" == Tier0* ]] &&  {`, so I expected it to be further indented.